scripts/actions/spells/songs/pining_nocturne.lua

```diff
@@ -0,0 +1,184 @@
+-----------------------------------
+-- Spell: Pining Nocturne
+-- Main effect: Increases the casting time of the target.
+-- Sub effect : Lowers target magic accuracy.
+-- Does not stack with Addle. Is overwrriten by Addle.
+-----------------------------------
+---@type TSpell
+local spellObject = {}
+
+spellObject.onMagicCastingCheck = function(caster, target, spell)
+    return 0
+end
+
+spellObject.onSpellCast = function(caster, target, spell)
+    -- https://www.bg-wiki.com/ffxi/Category:Nocturne
+    -- Current state may be a precursor to general BRD song handling.
+
+    -- Soul voice affects different songs differently.
+    local soulVoiceBoostType = {
```

Comment:
For clarity while reading these sections, you can extract these little tables of stuff (`strongAffinityAcc` too, etc.) to the parent scope

Reply:
Also, the style we use is
```lua
local tableName =
{
    Stuff
}